### PR TITLE
Added support for multiple instances of Android TV

### PR DESCRIPTION
**Pull request by albaintor**
Hi,

I have added support for multiple Android devices based on your work on AppleTV integration.
I have tested successfully the initialization of the integration with one device, then the addition of another AndroidTV device which gives me a second entity.

I have managed to create activities with both devices. However I haven't tried (yet) to send commands from one device or another as I was using the core simulator (which cannot send commands right ?)

I have also updated translations

### CODE REVIEW

**Review by zehnm — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Unfortunately this interferes with my current work in branch `27_new-feat-cmds`, which should not be merged into another PR! I've also already started with multi-device support.

I have to leave this PR open until my branch is ready and merged. Then we can check what we can cherry pick.

For the future please reach out first when doing bigger changes to avoid parallel work, see [CONTRIBUTING.md](https://github.com/unfoldedcircle/integration-androidtv/blob/main/CONTRIBUTING.md).

I highly suggest to create a dedicated branch for each PR in your repo and avoid using the main branch. Otherwise everything you push to main ends up in the PR! Rebasing and PR changes become much easier too.
Preferrably, a PR should only contain one feature in one commit. For example: one PR for translations, another for app link enhancements, and a third for multi-instance support.

File: .idea/misc.xml
[the remark this answers is not in the document]
Comment (albaintor) [reply .idea/misc.xml:1]: Ok reverted

File: intg-androidtv/apps.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "Pluto TV": {"url": "https://pluto.tv/"},
     "ARD Mediathek": {"url": "https://www.ardmediathek.de/"},
     "ZDF Mediathek": {"url": "https://www.zdf.de/filme"},
+    "Kodi": {"url": "https://play.google.com/store/apps/details?id=org.xbmc.kodi"}
```
[the remark this answers is not in the document]
Comment (albaintor) [reply intg-androidtv/apps.py:24]: I found on forums that it should work but I haven't been able to test it yet (see message after). However this is a mistake and I didn't intend to include this. But if it works why not...

**Comment by albaintor** [comment]
Hi again, my PR is incomplete : I just saw that the driver generate a pair certificate files androidtv_remote_cert.pem and androidtv_remote_key.pem -> I didn't pay attention. One pair of files per device is needed...

File: intg-androidtv/apps.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (albaintor) [reply intg-androidtv/apps.py:24]: It doesn't work, the kodi app is missing url scheme with deeplink to lauch the application. Only possible through adb at this state

**Comment by zehnm** [comment]
Closing: new PR submitted

**Pull request closed without merging**